readme.adoc

```diff
@@ -29,38 +29,35 @@ Table of contents overview:
 Your feedback and contributions are most welcome! Instructions for
 building and contributing can be found in <<appendixbuild>>.
 
+=== What to expect?
+
+Bitcoin is a huge subject and we couldn't possibly cover all of its aspects here. Nevertheless, we
+hope to discuss the necessary features to get you started as well as to enable you to further explore it on your own.
+
+There are lots of people involved in Bitcoin; as some of them have opposing opinions,
+there are resources that express contradictory ideas. However, we always
+attempt to stick to the facts, where opinions do not matter.
+
+=== Who wrote this?
+
+The main authors of this book are Kalle Rosenbaum and Linnéa Rosenbaum. This work
+was commissioned and funded by Chaincode Labs, a development centre that runs educational
```

Comment:
This brings up the question of what flavor of English we are using for the doc. You write organized with a Z below...  [Optech](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md) allows for a mix but is strict on some specific styling.

```suggestion
was commissioned and funded by Chaincode Labs, a development center that runs educational
```

Reply:
Right, I just assumed we were using American English but you tell me which rules to follow

Reply:
Paging @kallerosenbaum

Reply:
Heh, I thought we wrote in American English, so maybe that's what we should aim for. I'm not always able to tell the difference, so expect a mixture. You have some serious editing to do @MartaTurchetta 😱😱😱